test/integration_test.py

```diff
@@ -104,7 +118,7 @@ def test_sac_system(fast_config: dict, system_path: str) -> None:
 @pytest.mark.parametrize("env_name", discrete_envs)
 def test_discrete_env(fast_config: dict, env_name: str) -> None:
     """Test all discrete envs on random systems."""
-    system_path = random.choice(ppo_systems + q_learning_systems)
+    system_path = random.choice(ppo_systems + q_learning_systems + sable_systems)
```

Comment:
Can we remove this, because sable will take a long time to run, rather just test sable in its own test :pray: